doc/article.tex

```diff
@@ -296,12 +296,12 @@ \subsection{Underlying Likelihood}
 p(data|model) &=\int dType\, dLuminosity\, p(Photometry_o |  Luminosity, Cosmology,Host,Throughput)  \nonumber\\
 & p(Luminosity| Type
 , Host, Population) p(Type | Rates, Host)  \nonumber\\
-& \times \int dType\, p(Type_0|Type) p(Type|Rates, Host) \nonumber \\
+& \times \int dType\, p(Type_o|Type) p(Type|Rates, Host) \nonumber \\
 & \times p(Redshift_o, Host_o| Host),
 \label{like:eqn}
 \end{align}
 where $data=\{Photometry_o, Type_o, Redshift_o, Host_o\}$,
-$model=\{Cosmology, Populations, Rates\}$, and the independence
+$model=\{Cosmology, Populations, Rates, Throughput, Host\}$, and the independence
```

Comment:
@AlexGKim, I've modified the final model to include host and throughput, but I feel like host would be something we would integrate over and not external in the model. Thoughts?